docs/docs/testing.md

```diff
@@ -529,14 +529,6 @@ In the above examples, we can see expect doesn't preserve the floating numbers.
 
 > For cases where using decimal is not optimal, see the [Jest Expect documentation](https://jestjs.io/docs/expect) for other options and methods.
 
-#### DateTime
```

Comment:
Why did you remove this? It's still true, isn't it?

Reply:
I decided to keep the original text. We no longer have to convert it to a string in the tests you've fixed, but I still think this is valuable information that can come in handy in other situations. If you don't agree we can always fix/change it in another PR